RIRImageResizing/Classes/NSObject/RIRResizeImageOperation.h

```diff
@@ -0,0 +1,32 @@
+//
+//  RIRResizeImageOperation.h
+//  Pods
+//
+//  Created by Richard Reitzfeld on 11/21/16.
+//
+//
+
+#import "RIRResizeImageOperationParameters.h"
+
+#import <Foundation/Foundation.h>
+
+
+
+
+
+@interface RIRResizeImageOperation : NSObject
+
+#pragma mark - resizeParameters
+@property (nonatomic, readonly, strong, nullable) RIRResizeImageOperationParameters* resizeParameters;
```

Comment:
Both of these properties can actually be `nonnull`, since trying to create an instance of this class with the init method will return a nil instance if either parameter is null.